packages/edit-navigation/src/components/menu-selector/style.scss

```diff
@@ -0,0 +1,16 @@
+.edit-navigation__menu-selector__select-menu-button {
+	width: 100%;
+}
+
+.edit-navigation-menu-selector {
+	padding: $grid-unit-15;
+	@extend .edit-navigation-editor;
+	&-header {
```

Comment:
This concatenation style is generally avoided on the project because it makes it harder to search the codebase.